### ISSUE TITLE
Fix system check warnings.

### ISSUE THREAD

**Issue by garnertb** [FireCARES/firecares#182]
Fix these warnings that appear when running tests and manage.py commands.

```
WARNINGS:
firestation.PopulationClass0Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass1Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass2Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass3Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass4Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass5Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass6Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass7Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass8Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
firestation.PopulationClass9Quartile.id: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
```

**Comment by garnertb** [comment]
To reproduce ssh in and run: 
````shell
sudo su firecares
workon firecares
python manage.py syncdb
````

**Comment by garnertb** [comment]
Fixed in aa76e0a09c3770f9179088de0d0d660f6ce4868a.
